### ISSUE TITLE
Is it possible only allow to upload videos?

### ISSUE THREAD

**Issue by tennessine** [CMB2/CMB2#1445]
How to limit user select only videos and How to preview video

**Comment by tw2113** [comment]
I would check out https://github.com/CMB2/CMB2/wiki/Field-Types#file and/or https://github.com/CMB2/CMB2/wiki/Field-Types#file_list which includes arguments regarding previews and limiting by file type. I believe the query_args types may rely on mimetype.

**Comment by rawsta** [comment]
Yep. i got it working via query_args and setting the correct mime-type.
```
'query_args'   => [
		'type' => [
			'video/mp4',
			'video/mpeg',
			'video/x-mpeg',
			'video/avi',
			'video/msvideo',
			'video/x-msvideo',
		]
	], // Only video attachment
```
